documentation/source/BenchmarkingYoloNAS.md

```diff
@@ -10,8 +10,8 @@ Doing so teaches us how to properly benchmark YoloNAS and understand its full po
 
 The first step is to export our YoloNAS model to ONNX correctly. Two actions must be taken before we export our model to onnx:
 1. We need to replace our layers with "fake quantized" ones - this happens when we perform post-training quantization or quantization-aware training with SG.
-So nothing to worry about if you performed PTQ/QAT with SG and hold your newly exported ONNX checkpoint.
-2. We must call model.prep_model_for_conversion - this is essential as YoloNAS incorporates QARepVGG blocks. Without this call, the RepVGG branches will not be fused, and our model's speed will decrease significantly!
+So nothing to worry about if you performed PTQ/QAT with SG and hold your newly exported ONNX checkpoint. Beware that inference time in Pytorch is slower with such blocks - but will be faster once converted to the TRT Engine.
```

Comment:
I would make the `prep_model_for_conversion` nuance to be the first in the list. 
Since the `fake quantized` layers are relevant to quantized models only.

Reply:
done in bbf097e0bc30aaee7e6611382e3d6d5daa9ba5ea